packages/components/plh/plh-kids-kw/components/module-list-item/module-list-item.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from "@angular/core";
+import { FlowTypes } from "packages/data-models";
+import { TemplateBaseComponent } from "src/app/shared/components/template/components/base";
+import { TemplateTranslateService } from "src/app/shared/components/template/services/template-translate.service";
+import {
+  getBooleanParamFromTemplateRow,
+  getStringParamFromTemplateRow,
+} from "src/app/shared/utils";
+
+interface IModuleListItemParams {
+  /* TEMPLATE PARAMETER: "module_image_asset". The image attached to the module */
+  moduleImageAsset: string | null;
+  /* TEMPLATE PARAMETER: "text_transform". The format of the text on the module item. Default null */
+  textTransform: "capitalise" | "uppercase" | null;
+  /* TEMPLATE PARAMETER: "is_nav_item". Marks an item as navigable  in order to show the navigation image asset. Default "true"  */
+  navItem: boolean;
```

Comment:
Could this be removed as an explicit property and whether or not to show the nav image just be inferred from whether or not an asset is provided?

Reply:
@jfmcquade I'll use the `is-locked` boolean property given that `navItem` property was added to make the routing more customisable, otherwise if routing is based on the assets then even a locked module will route to a linked page when a `goto` action is defined, which is not desired.